lib/one_signal/subscription.ex

```diff
@@ -0,0 +1,105 @@
+defmodule OneSignal.Subscription do
+  use OneSignal.Entity
+  import OneSignal.Request
+
+  defp plural_endpoint(id) do
+    "/subscriptions" <> "/#{get_id!(id)}"
+  end
+
+  # defp plural_endpoint(%{alias_label: alias_label, alias_id: alias_id}) do
+  #   "/users/by/#{alias_label}" <> "/#{get_id!(alias_id)}" <> plural_endpoint()
+  # end
```

Comment:
Should be good to delete this?

Reply:
I would leave it there since the next ticket will use it. :D